### ISSUE TITLE
Why it cannot insert icons into table cell?

### ISSUE THREAD

**Issue by Fzoltan87** [josh18/TinyMCE-FontAwesome-Plugin#62]
Hi there!

I would like to ask you about awesome icon fetaure. Why it cannot insert icons into table cell?
I'm using a TinyMCE 4.7.12 version, and the awesome icons are inserted inside a <p></p> html tag only. Is there a olution if i wanted to insert icons insode a html table cell, please? Any help will be thank you very much! Best regards.

**Comment by josh18** [comment]
Not sure sorry but I am sure it used to work. Have you tried it using the `demo.html` file from this repo?

**Comment by Fzoltan87** [comment]
Yes, the icon is placed into the cell, but if i delete a non visible space (&nbsp;),which is located after the icon, then it clears the icon. Is there a way i can change the icon's color after i inserted it? Thank you for the help.
